CHANGELOG.md

```diff
@@ -734,6 +734,9 @@ Also, please note:
   * The bootstrap token system now allows token management and expiration
 * The [`kubefed` federation bootstrap tool](https://kubernetes.io/docs/tutorials/federation/set-up-cluster-federation-kubefed/) has also graduated to beta.
 * Interaction with container runtimes is now through the CRI interface, enabling easier integration of runtimes with the kubelet. Docker remains the default runtime via Docker-CRI (which moves to beta).
+  * **WARNING:** A [known issue](https://github.com/kubernetes/kubernetes/issues/44041)
```

Comment:
Did you make this a sub-item because the bug was triggered by the switch to CRI?